docs/CMakeLists.txt

```
@@ -0,0 +1,32 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+project(docs)
```

you would want to use `LANGUAGES NONE` to avoid testing for a C / C++ compiler when building the docs.

```suggestion
project(docs LANGUAGES NONE)
```

---

fixed